### ISSUE TITLE
Weighing any local path yields identical filesizes

### ISSUE THREAD

**Issue by mariuslundgard** [bjoerge/weigh#10]
Hey @bjoerge, thanks for a great module :-)

Maybe I'm doing something wrong, cause I can't get it to work correctly with local modules.

Weighing a local module like so:

```
$ weigh ./most-w3msg
Approximate weight of ./most-w3msg:
  Uncompressed: 454 B
  Minified (uglify): 448 B
  Minified and gzipped (level: default): 289 B
```

And weighing the same module via npm:

```
$ weigh most-w3msg
Approximate weight of most-w3msg:
  Uncompressed: 6.62 kB
  Minified (uglify): 1.64 kB
  Minified and gzipped (level: default): 681 B
```

Weighing any other local directory also yields `289 B`. Am I doing something wrong?

This is Node v6.3.0 and npm v3.10.6.

**Comment by bjoerge** [comment]
Hey @mariuslundgard! That is definitely a bug, I guess the 289 bytes is just for the browserify prelude, so probably no entry file gets resolved and included in the final bundle.

I'm currently refactoring the code base of `weigh` (it has become rather messy), and I'll try to fix this in the process. Thanks for reporting :)

**Comment by bjoerge** [comment]
Should be fixed in v3.3.5
